src/components/HeroesItem/heroesItem.scss

```diff
@@ -0,0 +1,41 @@
+.heroes {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  border-radius: 20px;
+  padding: 10px;
+
+  &__avatar {
+    width: 100px;
+    height: 100px;
+    background-color: red;
+    border-radius: 50%;
+      img {
+        border-radius: 50%;
+        width: 100%;
+        height: 100%;
+      }
+  }
+  
+  &__info {
+    margin-left: 40px;
+  }
+  &__btn {
```

Comment:
empty line is needed between 2 style blocks

Reply:
done